scss/partials/_publish.scss

```diff
@@ -7,3 +7,11 @@
       width: 100%;
     }
 }
+
+.popup-vertical-buttons .popup-buttons {
+  display: block;
+}
+.popup-vertical-buttons .popup-buttons .button{
```

Comment:
these classes should be within the ".publish" class, otherwise, the !important will affect the entire application when you use popup-buttons